content/articles/2019-02-dplyr-0-8-0.Rmarkdown

```diff
@@ -0,0 +1,154 @@
+has been tremendously useful in making the release process smoother. 
+
+The bulk of the changes are internal, and part of an ongoing effort to make the codebase more robust and 
+less surprising. This is an investment that will continue to pay off for years, and serve as a foundation for 
+more innovations in the future. 
+
+For a comprehensive list of changes, please see [the NEWS](https://dplyr.tidyverse.org/news/index.html#dplyr-0-8-0) 
+for the 0.8.0 release, the sections below discusses the main changes. 
+
+## Group hug
+
+Grouping has always been at the center of what `dplyr` is about, this release expands on the 
+existing `group_by()` with a set of *experimental* functions with a variety of 
+perspectives on the notion of grouping. 
+
+We believe they offer new unique possibilities, but we welcome community feedback and use cases
+before we put a `r emo::ji("ring")` on them. Let's illustrate them with a subset from the 
+well know `gapminder` data. 
+
+```{r}
+oceania <- gapminder::gapminder %>% 
+  filter(continent == "Oceania") %>% 
+  mutate(yr1952 = year - 1952) %>% 
+  select(-continent) %>% 
+  group_by(country)
+oceania
+```
+
+  - [group_nest()](https://dplyr.tidyverse.org/reference/group_nest.html) is similar to 
+  `tidyr::nest()` but focuses on the variables to *nest by* instead of the nested columns. 
+  
+```{r}
+oceania %>% 
+  group_nest()
+```
+  
+  - [group_split()](https://dplyr.tidyverse.org/reference/group_split.html) is a tidy version 
+  of `base::split()`. In particular it respects a `group_by()`-like grouping specification, 
+  and refuses to name its result. 
+  
+```{r}
+oceania %>% 
+  group_split()
+```
+  
+  - [group_map()](https://dplyr.tidyverse.org/reference/group_map.html) and [group_walk()](https://dplyr.tidyverse.org/reference/group_map.html) offer 
+  a way to iterate on groups of a grouped data frame. 
+  
+```{r}
+oceania %>% 
+  mutate(yr1952 = year - 1952) %>% 
+  group_map(~broom::tidy(stats::lm(lifeExp ~ yr1952, data = .x)))
+```
+  
+  - [group_data()](https://dplyr.tidyverse.org/reference/group_data.html), [group_rows()](https://dplyr.tidyverse.org/reference/group_data.html), and 
+  [group_keys()](https://dplyr.tidyverse.org/reference/group_split.html) expose the grouping information, that has 
+  been restructured in a tibble. 
+
+```{r}
+oceania %>% 
+  group_data()
+
+oceania %>% 
+  group_keys()
+
+oceania %>% 
+  group_rows()
+```
+ 
+  - [group_by()](https://dplyr.tidyverse.org/reference/group_by.html) gains a `.drop` argument
+  which you can set to `FALSE` to respect empty groups associated with factors, more on this below. 
+
+## Give factors some love
+
+The internal grouping algorithm has been redesigned to make it possible to 
+better respect factor levels and empty groups. To limit the disruption, we have not made
+this the default behaviour, you have to set [group_by()](https://dplyr.tidyverse.org/reference/group_by.html)'s 
```

Comment:
I would break the second part into its own sentence.
"To keep empty factor levels, you have to set…" (or however you want to phrase the beginning)